src/main/java/de/komoot/photon/utils/QueryToJson.java

```diff
@@ -16,12 +16,14 @@
 
     @Override
     public String convert(QueryBuilder anItem) {
-        try {
-            BytesReference bytes = anItem.toXContent(JsonXContent.contentBuilder(), new ToXContent.MapParams(null)).bytes();
-            return new String(bytes.toBytes(), "UTF-8");
-        } catch (IOException e) {
-            log.error("Unable to transform querybuilder to a json string due to an exception", e);
-            return null;
-        }
+	try {
+	    BytesReference bytes = anItem.toXContent(JsonXContent.contentBuilder(), new ToXContent.MapParams(null))
+		    .bytes();
+
+	    return bytes.utf8ToString();
+	} catch (IOException e) {
+	    log.error("Unable to transform querybuilder to a json string due to an exception", e);
+	    return null;
```

Comment:
I know this was already there but I think we should throw an exception here instead of silently accepting it, e.g. 
`throw new RuntimeException("Unable to transform ....", e);`